utensor_cgen/backend/transformer.py

```diff
@@ -0,0 +1,75 @@
+import pickle
+
+from utensor_cgen.logger import logger
+from utensor_cgen.transformer.pipeline import TransformerPipeline
+from utensor_cgen.utils import class_property
+
+from .base import BackendPart
+
+__all__ = ['PipelineTransformer']
+
+class PipelineTransformer(BackendPart):
+  TARGET = 'generic'
+  PART = 'pipeline_transformer'
+
+  def __init__(self, config):
+    self.transformer = TransformerPipeline(
+      methods=self.config['transform_methods']
+    )
+    self.trans_methods = self.config['transform_methods']
+    self.save_graph = self.config['save_graph']
+
+  def transform(self, ugraph):
+    logger.info("Transforming graph: %s", ugraph.name)
+    logger.info("Transform pipeline: %s", ' -> '.join(self.trans_methods))
+    self._check_non_quantized(ugraph)
+    new_ugraph = self.transformer.transform(ugraph)
+    new_ugraph.name = ugraph.name
+    logger.info('Graph transormation done')
+    if self.save_graph:
+      logger.info('Saving transformed graph')
+      pkl_fname = "{}_transformed.pkl".format(ugraph.name)
+      with open(pkl_fname, 'wb') as fid:
+        pickle.dump(new_ugraph, fid)
+      logger.info('{} saved'.format(pkl_fname))
+    return new_ugraph
+  
+  @classmethod
+  def _check_non_quantized(cls, ugraph):
```

Comment:
This method should check if input or output tensors have quantization params. It probably should not be tied to operator names

Reply:
Is it reasonable that we consider the model is quantized if there is any tensors in the model has quantization parameters?

Reply:
I'll rename it to `_check_generic` and do nothing.
Once we have a full list of generic ops, I'll fill this method body.